plugins/chat_error_monitor.py

```diff
@@ -9,10 +9,20 @@ class Coala_lowercase_c(BotPlugin):
 
     def callback_message(self, msg):
         emots = [':(', ':angry:', ':confounded:',
-                 ':disappointed:', ':triumph:']
+                 ':disappointed:', ':triumph:' , ':D']
```

Comment:
I don't think a smiley face emoji is a good option here. This is a list of emojis that will be randomly used down on line 30, and a smiley doesn't fit the pattern.

Reply:
In `match_coala` bot is printing random emoji from a list  of emots, changing emots is not a good idea ,there is also a 20% chance that  `:D`  will be shown if some one write `coala` with uppercase, 
The